audioled/effects.py

```diff
@@ -638,3 +638,200 @@ def process(self):
         config = self.displacement * math.sin(self._t * self.swingspeed)
 
         self._outputBuffer[0] = sp.ndimage.interpolation.shift(pixels, [0, config], mode='wrap', prefilter=True)
+
+
+class Flipping(Effect):
+    """Effect that flips output array."""
+
+    @staticmethod
+    def getEffectDescription():
+        return \
+            "Flipping output. Useful for multi-outputs."
+
+    def __init__(self, Flip=True):
+        self.Flip = Flip
+        self.__initstate__()
+
+    def __initstate__(self):
+        # state
+        super(Flipping, self).__initstate__()
+
+    @staticmethod
+    def getParameterDefinition():
+        definition = {
+            "parameters":
+            OrderedDict([
+                ("Flip", True),
+            ])
+        }
+        return definition
+
+    @staticmethod
+    def getParameterHelp():
+        help = {
+            "parameters": {
+                "Flip": "Activate the Flip."
+            }
+        }
+        return help
+
+    def getParameter(self):
+        definition = self.getParameterDefinition()
+        definition['parameters']['Flip'] = self.Flip
+        return definition
+
+    def numInputChannels(self):
+        return 1
+
+    def numOutputChannels(self):
+        return 1
+
+    def process(self):
+        if self._inputBuffer is None or self._outputBuffer is None:
+            return
+        if not self._inputBufferValid(0):
+            self._outputBuffer[0] = None
+            return
+
+        pixels = self._inputBuffer[0]
+
+        if self.Flip is True:
```

Comment:
Flipping can be done in 1 operation using the axis, see https://numpy.org/doc/stable/reference/generated/numpy.flip.html